rudder-lang/src/bin/main.rs

```diff
@@ -48,32 +67,60 @@ struct Opt {
     #[structopt(long, short)]
     input: PathBuf,
     /// Set to use technique translation mode
-    #[structopt(long)]
+    #[structopt(long, short)]
     translate: bool,
     /// Set to compile a single technique
-    #[structopt(long)]
-    technique: bool,
+    #[structopt(long, short)]
+    compile: bool,
     /// Output format to use
     #[structopt(long, short = "f")]
-    output_format: Option<String>,
+    output_fmt: Option<String>,
```

Comment:
Couldn't the default value be handled by structuopt?

Reply:
Well as of now output_format/fmt is not called anywhere but as soon as it is, I will update it depending on what we need it to be

Reply:
I commented on the wrong line :man_facepalming: